MaterialLib/PorousMedium/UnsaturatedProperty/CapillaryPressure/VanGenuchtenCapillaryPressureSaturation.h

```diff
@@ -78,6 +86,66 @@ class VanGenuchtenCapillaryPressureSaturation final
 private:
     const double _pb;  ///< Entry pressure.
     const double _m;   ///< Exponent m, m in [0,1]. n=1/(1-m).
+    bool _has_regularized;
```

Comment:
You can add a formula about this regularized  model in the class documentation (see lines 23--43).